docs/USAGE-GUIDE.md

```diff
@@ -561,6 +561,19 @@ int s2n_config_add_cert_chain_and_key_to_store(struct s2n_config *config,
 
 **s2n_config_add_cert_chain_and_key_to_store** may be called multiple times to support multiple key types(RSA, ECDSA) and multiple domains. On the server side, the certificate selected will be based on the incoming SNI value and the client's capabilities(supported ciphers). In the case of no certificate matching the client's SNI extension or if no SNI extension was sent by the client, the certificate from the **first** call to **s2n_config_add_cert_chain_and_key_to_store** will be selected.
 
+### s2n\_config\_set\_cert\_chain\_and\_key\_defaults
+
+```c
+int s2n_config_set_cert_chain_and_key_defaults(struct s2n_config *config,
+                                               struct s2n_cert_chain_and_key **list_cert_key_pairs,
+                                               uint8_t num_defaults_to_set);
```

Comment:
nit: I don't expect there to ever be more than 256 defaults but I don't see any cost in making the argument a larger size. uint32_t?